src/app/parcels/parcelsTable/ParcelsPage.tsx

```diff
@@ -323,6 +323,7 @@ const ParcelsPage: React.FC<{}> = () => {
                                 sortableColumns: parcelsSortableColumns,
                                 setSortState: setSortState,
                             }}
+                            defaultSortField={defaultParcelsSortKey}
```

Comment:
This `defaultSortField` and underlying Table.tsx changes are to resolve the outstanding request for visible default sorting on the Parcels page as per this comment: https://github.com/NorwoodAndBrixtonFoodbank/nbf-website/issues/152#issuecomment-2154830486